docs/odh/grafana/add_grafana_dashboard.md

```diff
@@ -0,0 +1,37 @@
+# Add a grafana dashboard to OperateFirst Grafana
+
+To be able to include your dashboards in Grafana, you will need to add a `GrafanaDashboard` custom resource. This custom resource is defined as a yaml file that can remotely reference the dashboard contents from a specified URL and falls back to embedded json if the URL can not be resolved.
+
+An example `grafana-dashboard-example.yaml` template is shown below:
+
+```yaml
+---
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: grafana-dashboard-from-url
+  labels:
+    app: grafana
+spec:
+  name: grafana-dashboard-from-url.json
+  url: https://raw.githubusercontent.com/integr8ly/grafana-operator/master/deploy/examples/remote/grafana-dashboard.json
+  json:
+```
+
+In this template, all you need to do is update:
+1. `metadata.name` - the dashboard name
+2. `spec.name` - the dashboard JSON file name
+3. `spec.url` - the URL pointing to the location of your dashboard JSON file
+
+***Note:** Ensure that the `spec.json` field is empty if setting the URL, if both url and json are specified then the json field won't be updated when fetched.*
```

Comment:
See above. I would avoid specifying both `url` and `json` fields at the same time. Let's either inline it (`json`) or have an `url` without a "magic" empty `json` field fallback.